supporting-blog-content/customer-success-example/README.md

```diff
@@ -0,0 +1,35 @@
+Example of AI Conversational Search using Elasticsearch, OpenAI and Vercel. 
+
```

Comment:
It might be useful to have a screenshot showing what the application looks like [optional but useful]